ccm_web/client/src/components/CustomTable.tsx

```diff
@@ -73,7 +73,6 @@ function CustomTable<T extends TableEntity> (props: TableProps<T>): JSX.Element
           <TableRow>
             <TablePagination
               rowsPerPageOptions={[rowsPerPage]}
-              colSpan={3}

```

Comment:
This was not an issue previously, because we didn't have any tables greater than 3 columns wide. `colSpan` does not seem to be present in the documentation for Material UI 4 or 5, but it still works? https://v4.mui.com/api/table-pagination/#tablepagination-api

Anyways, removing it seems to trigger a default that works in all uses cases.